OpenRA.Mods.Common/Traits/GivesExperience.cs

```diff
@@ -41,14 +41,16 @@ public void Killed(Actor self, AttackInfo e)
 
 			var valued = self.Info.TraitInfoOrDefault<ValuedInfo>();
 
-			// Default experience is 100 times our value
 			var exp = info.Experience >= 0
 				? info.Experience
-				: valued != null ? valued.Cost * 100 : 0;
```

Comment:
Actually, now that I think of it, having that upgrade rule is a terrible solution here. That change sacrifices the possibility to have levelup requirements like 2.5 times the cost.

A better option would be to change https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Mods.Common/Traits/GainsExperience.cs#L61 to `kv.Key * value / 100` without touching the yamls.